src/main/scala/bench/Benchmark.scala

```diff
@@ -0,0 +1,239 @@
+package bench
```

Comment:
I suppose it's not really important here because we don't expect to publish any artifacts. In general you don't want to have test or benchmarking code be part of the main project / artifact to avoid cluttering the source file structure and introducing accidental dependencies.